view/frontend/web/css/styles.css

```diff
@@ -94,4 +94,9 @@
 }
 #omise_offsite_internetbankingForm img.kbank, #omise_offsite_installmentForm img.kbank {
     background: url('../images/kbank.svg')rgb(247, 226, 226);
-}
\ No newline at end of file
+}
+
+#omise_offsite_truemoneyphoneNumber {
+    width: 225px;
+    margin-bottom:18px;
+}
```

Comment:
These numbers may affect with the mobile or tablet screen, would be nice if we also test on those 2 screens if it fits properly.

Reply:
that `width` style is copied from credit card, margin-bottom looks normal on the phone.